jdk/src/windows/bin/amd64/jvm.cfg

```diff
@@ -34,5 +34,10 @@
 # "-XXaltjvm=<jvm_dir>" option, but that too is unsupported
 # and may not be available in a future release.
 #
--server KNOWN
+-j9vm KNOWN
+-hotspot IGNORE
```

Comment:
You can remove the IGNORE for hotspot, classic, native, and green, since they didn't exist before.

Reply:
done